test/16_recode.jl

```diff
@@ -1,14 +1,41 @@
 module TestRecode
 using Test
 using CategoricalArrays
-using CategoricalArrays: DefaultRefType
+using CategoricalArrays:DefaultRefType
 
 if VERSION < v"0.7.0-"
-    using CategoricalArrays: replace!
+    using CategoricalArrays:replace!
 end
 
 const ≅ = isequal
 
+function test_recode_performance(orig_vec, cat2merge_vec)
```

Comment:
This would be more appropriate in the benchmarks folder (see how to use the PkgBenchmark/BenchmarkTools syntax there).

Reply:
Ok but having just a benchmark provides weaker protection against an accidental performance degradation as opposed to performance test (because the benchmarks are not integrated into CI).

Reply:
Yes but CI machines cannot reliably be used to check performance. They may have unexpected slowdowns, while BenchmarkTools uses many runs (normally on a dedicated machine) to avoid this kind of problem.